ddtrace/internal/serverless/__init__.py

```diff
@@ -17,3 +17,14 @@ def has_aws_lambda_agent_extension():
     extension available.
     """
     return path.exists("/opt/extensions/datadog-agent")
+
+
+def in_gcp_function():
+    # type: () -> bool
+    """Returns whether the environment is a GCP Function.
+    This is accomplished by checking for the presence of one of two pairs of environment variables,
+    with one pair being set by deprecated GCP Function runtimes, and the other set by newer runtimes.
+    """
+    is_deprecated_gcp_function = bool(environ.get("FUNCTION_NAME", False)) and bool(environ.get("GCP_PROJECT", False))
+    is_newer_gcp_function = bool(environ.get("K_SERVICE", False)) and bool(environ.get("FUNCTION_TARGET", False))
```

Comment:
nit: we don't need to convert to bool, we can just check if the environment variable does not contain an empty string.
```suggestion
    is_deprecated_gcp_function = environ.get("FUNCTION_NAME", "")  != "" and environ.get("GCP_PROJECT",  "")  != ""
    is_newer_gcp_function = environ.get("K_SERVICE", "")  != "" and environ.get("FUNCTION_TARGET", "")  != ""
```